crates/ika-core/src/dwallet_mpc/mpc_manager.rs

```diff
@@ -772,12 +622,129 @@ impl DWalletMPCManager {
         self.mpc_sessions.insert(*session_identifier, new_session);
     }
 
-    pub(super) async fn must_get_next_active_committee(&self) -> Committee {
-        self.next_epoch_committee_receiver
-            .clone()
-            .wait_for(|committee| committee.epoch == self.epoch_id + 1)
```

Comment:
I don't like this - we `must wait`? and block? why, we can simply get it when it comes, and do other stuff in the meantime.